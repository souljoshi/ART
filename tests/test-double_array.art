// This test is to see if arrays support doubles.

// include doubles in the array
int main()
{	

	double [5] x;
	int i;
	string s;
	s = " ";

	x[0] = 1.777777;
	x[1] = 2.777777;
	x[2] = 3.7;
	x[3] = 4.777;
	x[4] = 5.7777;
	for (i = 0; i<5; i += 1){
		printf(x[i]);
		//prints(s);
	}
}