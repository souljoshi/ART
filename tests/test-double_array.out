1.7777772.7777773.7000004.7770005.777700